net.sf.eclipsecs.checkstyle/pom.xml

```diff
@@ -9,40 +9,28 @@
     <artifactId>net.sf.eclipsecs.checkstyle</artifactId>
     <packaging>eclipse-plugin</packaging>
     <name>Checkstyle Core Library Plugin</name>
-    
+
     <properties>
         <checkstyle.version>10.10.0</checkstyle.version>
     </properties>
     <build>
         <plugins>
             <plugin>
-                <groupId>com.googlecode.maven-download-plugin</groupId>
-                <artifactId>download-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>download-checkstyle-src</id>
-                        <phase>initialize</phase>
-                        <goals>
-                            <goal>wget</goal>
-                        </goals>
-                        <configuration>
-                            <outputDirectory>.</outputDirectory>
-                            <url>https://github.com/checkstyle/checkstyle/archive/checkstyle-${checkstyle.version}.zip</url>
-                            <outputFileName>checkstyle-checkstyle-${checkstyle.version}.zip</outputFileName>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>        
-            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
                     <execution>
-                        <phase>prepare-package</phase>
+                        <id>repackage-checkstyle-source</id>
+                        <phase>generate-sources</phase>
                         <configuration>
                             <target>
+                                <get src="https://github.com/checkstyle/checkstyle/archive/checkstyle-${checkstyle.version}.zip"
+                                     dest="${basedir}/checkstyle-${checkstyle.version}.zip"
```

Comment:
Shouldn't this really go to the target folder instead? Doesn't base dir put it into the path of what gets cataloged for git, so this would show up as an unchanged file?

Reply:
Completely right, and I was thinking the same. However, it has been like that since eternity. I wasn't sure if I would break some of the scripts of calixte when moving this around, therefore I would like to do that as additional cleanup later (also looking at other places where files might be created outside the maven target directory).